charts/latest/azuredisk-csi-driver/templates/csi-azuredisk-node-windows-hostprocess.yaml

```diff
@@ -129,6 +129,9 @@ spec:
             - "--get-nodeid-from-imds={{ .Values.node.getNodeIDFromIMDS }}"
             - "--enable-otel-tracing={{ .Values.windows.otelTracing.enabled }}"
             - "--metrics-address=0.0.0.0:{{ .Values.node.metricsPort }}"
+{{- if eq .Values.node.removeNotReadyTaint false }}
```

Comment:
remove this line

Reply:
Acked and thanks, you mean display declaration parameters even though set to true we could omit it, right?

Reply:
```
- "--remove-not-ready-taint={{ .Values.node.removeNotReadyTaint }}"
```
is enough, don't need L132

Reply:
sure, done.